cluster/gce/manifests/kube-apiserver.manifest

```diff
@@ -45,7 +45,7 @@
         "scheme": "HTTPS",
         "host": "127.0.0.1",
         "port": {{secure_port}},
-        "path": "/healthz"
+        "path": "/readyz?exclude=etcd&exclude=kms-provider-0&exclude=kms-provider-1"
```

Comment:
I'm surprised we would consider an API server ready when these checks were failing

Reply:
I think the crux of the issue is that readyz probes will keep restarting kube-apiserver if etcd or kms-plugin are not ready. Since doing so is unlikely to address issues within the external dependencies of the AP server, it probably makes sense to remove these checks from probes' configuration.

External entities (ex. external health monitoring services or load balancers) could still use this endpoint to determine the "true" readiness of the API server.

Reply:
Failure of readiness probes will not cause restarts of the kube-apiserver.

Reply:
You are right, I should have read our docs more carefully. The behavior for the readiness probe is different than that of the liveness probe.

[failureThreshold](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-readiness-probes):  In case of readiness probe the Pod will be marked Unready. 

What I am not super clear about is what "marked as Unready" means in the context of a static pod.